src/scenes/Event/store.js

```diff
@@ -118,6 +118,30 @@ export default class EventStore {
     this.setReactions();
   }
 
+  @action
+  async initTxfees() {
```

Comment:
Why do you need this? Tx cost should be calculated when the user does the action. And this one is assuming we are doing a `bet`, but the EventStore is used for all types of txs. 

It is also unused, remove it.